DSharpPlus/Entities/Interaction/Components/DiscordSelectComponent.cs

```diff
@@ -33,12 +33,6 @@ namespace DSharpPlus.Entities
     /// </summary>
     public sealed class DiscordSelectComponent : DiscordComponent
     {
-        /// <summary>
-        /// The custom Id of this component.
-        /// </summary>
-        [JsonProperty("custom_id", NullValueHandling = NullValueHandling.Ignore)]
-        public new string CustomId { get; set; }
```

Comment:
Is this because the custom id is already defined in DiscordComponent?

Reply:
Yes.